cpp/src/neighbors/ivf_pq/ivf_pq_build_common.cu

```diff
@@ -1,9 +1,9 @@
 /*
- * raft::copyright (c) 2022-2024, NVIDIA CORPORATION.
+ * Copyright (c) 2022-2024, NVIDIA CORPORATION.
```

Comment:
I didn't notice this one before. That's a great search and replace snafu. This one just made my day.